src/sonic-config-engine/minigraph.py

```diff
@@ -229,6 +229,16 @@ def parse_dpg(dpg, hname):
         vlan_intfs = []
         vlans = {}
         vlan_members = {}
+        intf_vlan_mbr = {}
+        for vintf in vlanintfs.findall(str(QName(ns, "VlanInterface"))):
+            vlanid = vintf.find(str(QName(ns, "VlanID"))).text
+            vintfmbr = vintf.find(str(QName(ns, "AttachTo"))).text
+            vmbr_list = vintfmbr.split(';')
+            for i, member in enumerate(vmbr_list):
+                try:
```

Comment:
from collections import defaultdict
intf_vlan_mbr = defaultdict(list)
intf_vlan_mbr[member].append[vlanid]

should work without the need for try/except blocks.